008_team_6_agent_design/008_team_6_agent_design.tex

```diff
@@ -1 +1,1014 @@
+
+
+%%%
+\subsubsection{Simulations D: Destabilisation of a Collectivist System}
+
+In these experiments, we initialise the tower's population with collectivist agents only however, unlike with A (\Cref{sec:simA}), these agents have the possibility to change their social motive as turns progress. 
+
+The goal of these experiments is to evaluate if a society comprised solely of collectivists is able to remain stable over time, noting that if this is achievable, we expect to see similar results to the ones presented in \Cref{fig:res_A} (c) and (d). In addition, we investigate the effect of treaties on such a system. The simulation results are shown in \Cref{fig:res_D}.
+
+\begin{figure}[H]
+    \centering
+    \begin{tabular}{ccc}
+    \includegraphics[width=0.3\linewidth]{008_team_6_agent_design/D/D1_SM.pdf} &   \includegraphics[width=0.3\linewidth]{008_team_6_agent_design/D/D1_deaths.pdf} &
+    \includegraphics[width=0.3\linewidth]{008_team_6_agent_design/D/D1_utility.pdf}\\[0pt]
+    (a) D1: social motives & (b) D1: deaths over time & (c) D1: utility over time \\[8pt]
+    %      \includegraphics[width=0.3\linewidth]{D/D2_SM.pdf} &   \includegraphics[width=0.3\linewidth]{D/D2_deaths.pdf} &
+    % \includegraphics[width=0.3\linewidth]{D/D2_utility.pdf}\\[0pt]
+    % (d) D2: social motives & (e) D2: deaths over time & (f) D2: utility over time \\[8pt]
+    \includegraphics[width=0.3\linewidth]{008_team_6_agent_design/D/D3_SM.pdf} &   \includegraphics[width=0.3\linewidth]{008_team_6_agent_design/D/D3_deaths.pdf} &
+    \includegraphics[width=0.3\linewidth]{008_team_6_agent_design/D/D3_utility.pdf}\\[0pt]
+    (d) D2: social motives & (e) D2: deaths over time & (f) D2: utility over time \\[8pt]
+    \end{tabular}
+    \caption{Simulation results different treaties acceptances.}
+    \label{fig:res_D}%
+\end{figure}
+
+The analysis of experiment D1 can be done as follows: during the first days in the tower, the agents will agree on collectivist treaties as introduced in \Cref{proposing_treaties}. As the turns progress and their health decreases, some agents change their social motive from collectivist to selfish (\Cref{fig:res_D} (a)). However, once their health falls below the \textit{weakLevel}, the treaties the agents signed whilst being collectivist no longer apply. This is because, as explained in \Cref{proposing_treaties}, the treaties proposed by collectivists only hold under the condition \textit{currentHP} $>$ \textit{WeakLevel}. Therefore, the selfish agents, now unconstrained, consume more food than what a collectivist agent normally would. This wastes utility, ultimately leading to a large amount of deaths at the end of day 10 (\Cref{fig:res_D} (b)). This result is similar to the one discussed in \Cref{simulation_C}.
```

Comment:
```suggestion
The analysis of experiment D1 can be done as follows: during the first days in the tower, the agents will agree on collectivist treaties as introduced in \Cref{proposing_treaties}. As the turns progress and their health decreases, some agents change their social motive from collectivist to selfish (\Cref{fig:res_D} (a)). However, once their health falls below the \texttt{weakLevel}, the treaties the agents signed whilst being collectivist no longer apply. This is because, as explained in \Cref{proposing_treaties}, the treaties proposed by collectivists only hold under the condition \texttt{currentHP} $>$ \texttt{WeakLevel}. Therefore, the selfish agents, now unconstrained, consume more food than what a collectivist agent normally would. This wastes utility, ultimately leading to a large amount of deaths at the end of day 10 (\Cref{fig:res_D} (b)). This result is similar to the one discussed in \Cref{simulation_C}.
```